userspace/falco/app_actions/print_support.cpp

```diff
@@ -37,13 +37,36 @@ application::run_result application::print_support()
 		nlohmann::json support;
 		struct utsname sysinfo;
 		std::string cmdline;
+		std::unique_ptr<sinsp> s(new sinsp());
+		char driver_api_version_string[32], driver_schema_version_string[32];
 
 		if(uname(&sysinfo) != 0)
 		{
 			return run_result::fatal(string("Could not uname() to find system info: ") + strerror(errno));
 		}
 
 		support["version"] = FALCO_VERSION;
+
+		support["libs_version"] = FALCOSECURITY_LIBS_VERSION;
+		support["plugin_api_version"] = s->get_plugin_api_version();
+		
+		auto driver_api_version = s->get_scap_api_version();
+		unsigned long driver_api_major = PPM_API_VERSION_MAJOR(driver_api_version);
+		unsigned long driver_api_minor = PPM_API_VERSION_MINOR(driver_api_version);
+		unsigned long driver_api_patch = PPM_API_VERSION_PATCH(driver_api_version);
+		snprintf(driver_api_version_string, sizeof(driver_api_version_string), "%lu.%lu.%lu", driver_api_major, driver_api_minor, driver_api_patch);
+
+		support["driver_api_version"] = driver_api_version_string;
+
+		auto driver_schema_version = s->get_scap_schema_version();
+		unsigned long driver_schema_major = PPM_API_VERSION_MAJOR(driver_schema_version);
+		unsigned long driver_schema_minor = PPM_API_VERSION_MINOR(driver_schema_version);
+		unsigned long driver_schema_patch = PPM_API_VERSION_PATCH(driver_schema_version);
+		snprintf(driver_schema_version_string, sizeof(driver_schema_version_string), "%lu.%lu.%lu", driver_schema_major, driver_schema_minor, driver_schema_patch);
```

Comment:
it would be amazing if we had a sinsp API to do that instead of using low levels macro, but we can do that in a next step :)

Reply:
Completely agree, since I wrote this when libs 0.10.0 was already about to be released I didn't want to add an API dependency right now, as harmless as it may be.